test/fail_compilation/retscope2.d

```diff
@@ -162,3 +162,67 @@ void foo800()
     scope Exception e;
     throw e;
 }
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
I guess it won't help avoid merge conflicts (you could insert the different test cases in separate places instead), but if it makes life easier for you, let's go ahead.